controllers/connector_controller.go

```diff
@@ -182,7 +181,7 @@ func (r *ConnectorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		logger.Error(err, "Failed to update Connector", "Kind", workLoadType, "Namespace", obj.GetNamespace(), "Name", obj.GetName())
 		return ctrl.Result{RequeueAfter: cons.DefaultRequeueIntervalInSecond}, err
 	}
-	return ctrl.Result{}, err
+	return ctrl.Result{RequeueAfter: cons.DefaultRequeueIntervalInSecond}, nil
```

Comment:
why requeue

Reply:
The update operation may fail due to abnormal container stop, so successful connectors also need to be re queued. The k8s controller component also has the same processing logic for pods

Reply:
remove requeue